metadata/org.apache.shardingsphere.elasticjob/elasticjob-lite-core/3.0.2/reflect-config.json

```diff
@@ -0,0 +1,2632 @@
+      },
+      {
+        "name": "setInstanceId",
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      },
+      {
+        "name": "setInstanceName",
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      },
+      {
+        "name": "setMisfireThreshold",
+        "parameterTypes": [
+          "long"
+        ]
+      }
+    ]
+  },
+  {
+    "condition": {
+      "typeReachable": "org.apache.shardingsphere.elasticjob.lite.api.bootstrap.impl.OneOffJobBootstrap"
+    },
+    "name": "org.quartz.simpl.RAMJobStoreBeanInfo"
+  },
+  {
+    "condition": {
+      "typeReachable": "org.apache.shardingsphere.elasticjob.lite.api.bootstrap.impl.OneOffJobBootstrap"
+    },
+    "name": "org.quartz.simpl.RAMJobStoreCustomizer"
+  },
+  {
+    "condition": {
+      "typeReachable": "org.apache.shardingsphere.elasticjob.lite.api.bootstrap.impl.OneOffJobBootstrap"
+    },
+    "name": "org.quartz.simpl.SimpleThreadPool",
+    "queryAllPublicMethods": true
+  },
+  {
+    "condition": {
+      "typeReachable": "org.apache.shardingsphere.elasticjob.lite.api.bootstrap.impl.ScheduleJobBootstrap"
+    },
+    "name": "org.quartz.simpl.SimpleThreadPool",
+    "queryAllPublicMethods": true
+  },
+  {
+    "condition": {
+      "typeReachable": "org.apache.shardingsphere.elasticjob.lite.internal.schedule.JobScheduler"
+    },
+    "name": "org.quartz.simpl.SimpleThreadPool",
+    "methods": [
+      {
+        "name": "<init>",
+        "parameterTypes": []
+      },
+      {
+        "name": "setInstanceId",
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      },
+      {
+        "name": "setInstanceName",
+        "parameterTypes": [
+          "java.lang.String"
+        ]
+      },
+      {
+        "name": "setThreadCount",
+        "parameterTypes": [
+          "int"
+        ]
+      }
+    ]
+  },
+  {
+    "condition": {
+      "typeReachable": "org.apache.shardingsphere.elasticjob.lite.api.bootstrap.impl.OneOffJobBootstrap"
+    },
+    "name": "org.quartz.simpl.SimpleThreadPoolBeanInfo"
+  },
+  {
+    "condition": {
+      "typeReachable": "org.apache.shardingsphere.elasticjob.lite.api.bootstrap.impl.OneOffJobBootstrap"
+    },
+    "name": "org.quartz.simpl.SimpleThreadPoolCustomizer"
+  },
+  {
+    "condition": {
+      "typeReachable": "org.apache.shardingsphere.elasticjob.executor.ElasticJobExecutor"
+    },
+    "name": "sun.misc.Unsafe",
+    "allDeclaredFields": true
+  },
+  {
+    "condition": {
+      "typeReachable": "org.quartz.simpl.SimpleJobFactory"
+    },
+    "name": "org.apache.shardingsphere.elasticjob.lite.internal.schedule.LiteJob",
+    "queryAllPublicMethods": true,
+    "methods": [
+      {
+        "name": "<init>",
+        "parameterTypes": []
+      },
+      {
+        "name": "setJobExecutor",
+        "parameterTypes": [
+          "org.apache.shardingsphere.elasticjob.executor.ElasticJobExecutor"
+        ]
+      }
+    ]
+  }
```

Comment:
- The typeReachable attribute of this condition metadata entry seems to be no more suitable than the `org.quartz.simpl.SimpleJobFactory` class, I added it manually.